docs/3.api/4.commands/init.md

```diff
@@ -38,7 +39,7 @@ Option | Default | Description
 `--gitInit` |  | Initialize git repository
 `--shell` |  | Start shell after installation in project directory
 `--packageManager` |  | Package manager choice (npm, pnpm, yarn, bun)
-`--modules` |  | Nuxt modules to install (comma separated without spaces)
+`-M, --modules` |  | Nuxt modules to install (comma separated without spaces)
 `--no-modules` |  | Skip module installation prompt
```

Comment:
We actually support `--no-M` (and more https://github.com/unjs/citty/issues/209)